### ISSUE TITLE
Meraki fails to locate `npm` if NOT installed with `nvm` on OSx

### ISSUE THREAD

**Issue by marcopiraccini** [platformatic/meraki#144]
In the case user gets:

```
2024-01-16T15:08:36.339Z - INFO - Installing @platformatic/service on /Users/antonio/Documents/meraki-test/test-draft-0-4-1...
2024-01-16T15:08:36.343Z - ERROR - zsh:1: command not found: npm
2024-01-16T15:08:36.350Z - ERROR - Error: Command failed with exit code 127: /bin/zsh -c . ~/.zshrc npm install @platformatic/service
zsh:1: command not found: npm
```
`npm` is installed but the path is not set on `.zshrc`: 
```
antonio@MBP-di-Antonio ~ % which npm 
/usr/local/bin/npm
antonio@MBP-di-Antonio ~ %
```

meraki should check if it can find `npm` sourcing `,zshrc` and if not, try `/usr/local/bin/npm`

**Comment by marcopiraccini** [comment]
We might have a list of "tentative" locations to look for `npm` if not in path.

**Comment by leorossi** [comment]
Closed by #163
